lib/internal/test_runner/runner.js

```diff
@@ -32,6 +34,10 @@ const {
 const { basename, join, resolve } = require('path');
 const { once } = require('events');
 
+const kMinPort = 1024;
```

Comment:
why is this the minimum port? if a user is running as root, they can use all the ports.

Reply:
this doesn't validate if the port is usable, it is only used to find the next port to use

Reply:
right. but ports under 1024 may be available to use

Reply:
node does not support debugging in those ports, I see no reason why this should behave differently
https://github.com/nodejs/node/blob/81d9cdb8cd5109f1beadf78558ff9e37b4c63473/src/node_process_object.cc#L66-L70
https://github.com/MoLow/node/blob/ae799bdf1a1c2576bcd1f17430bc8f5f1997a631/src/node_options.cc#L967

Reply:
ah ok, i didn't realize node's debugger was restricted in that fashion.

perhaps there could be shared constants for these two values rather than hardcoding them in two places?

Reply:
done, added a TODO to move the implementation of `cluster` to use that utility as well